src/lib/requests.ts

```diff
@@ -30,17 +30,20 @@
  --------------
  ******/
 /* istanbul ignore file */
-// Testing will be covered in PR #11
+// Testing will be covered in Ticket #354
 
 import { config } from './config'
+import { ThirdpartyRequests, BaseRequestConfigType } from '@mojaloop/sdk-standard-components'
+import Logger from '@mojaloop/central-services-logger'
 // @ts-ignore
```

Comment:
Remove `@ts-ignore` and extra comment in line 39.

Reply:
addressed